route/build/src/network_topo.f90

```diff
@@ -548,6 +551,7 @@ SUBROUTINE REACH_LIST(&
  integer(i4b)      , intent(out)                :: ierr            ! error code
  character(*)      , intent(out)                :: message         ! error message
  ! local variables
+ real(dp), parameter                            :: verySmall=tiny(1.0_dp) ! a very small number
```

Comment:
verySmall is defined in public.f90.   

Now, not clear what to be stored in public.f90, globalData.f90.    maybe have to have some rule.

Reply:
Fixed